environment/swarmball_env.py

```diff
@@ -28,6 +35,7 @@ def step(self, action):
     def reset(self):
         self.thresh_vel = [0 for _ in range(self.cluster_count)]
         self.sim.reset()
+        self.goal_prev_pos = self.sim._goal_object.body.position[0]
 
```

Comment:
Może od razu dodaj w tym PR tego returna? Chyba nie ma sensu robić osobnej karteczki na to.

Reply:
Done.